Packages/vcs/Lib/vtk_ui/line.py

```diff
@@ -0,0 +1,115 @@
+import vtk
+
+class Line(object):
```

Comment:
I needed a line to be a blinking cursor for the `Textbox` object, and wound up writing this wrapper for `vtkPolyDataMapper2D` and `vtkLineSource` so I didn't need to fuss with handling VTK stuff while I was beating my brains out on getting the cursor to be in the right place.